full/Dockerfile

```
@@ -190,7 +195,14 @@ RUN cp /home/gitpod/.profile /home/gitpod/.profile_orig && \
     && .cargo/bin/rustup completions bash cargo | sudo tee /etc/bash_completion.d/rustup.cargo-bash-completion > /dev/null \
     && grep -v -F -x -f /home/gitpod/.profile_orig /home/gitpod/.profile > /home/gitpod/.bashrc.d/80-rust
 ENV PATH=$PATH:$HOME/.cargo/bin
+# TODO: setting CARGO_HOME to /workspace/.cargo avoids manual updates. Remove after full workspace backups are GA.
+ENV CARGO_HOME=/workspace/.cargo
+ENV PATH=$CARGO_HOME/bin:$PATH
 
+USER root
+RUN mkdir -p $CARGO_HOME \
+    && chown -R gitpod.gitpod $CARGO_HOME
+USER gitpod
```

Can be simplified. Also, shouldn't it be `gitpod:gitpod` instead of `gitpod.gitpod`?

```suggestion
RUN sudo mkdir -p $CARGO_HOME \
    && sudo chown -R gitpod:gitpod $CARGO_HOME
```

---

done